include/AutomationEditor.h

```diff
@@ -26,16 +26,18 @@
 #ifndef LMMS_GUI_AUTOMATION_EDITOR_H
 #define LMMS_GUI_AUTOMATION_EDITOR_H
 
+#include <QPushButton>
 #include <QWidget>
 #include <array>
 
+#include "AutomationClip.h"
+#include "ComboBoxModel.h"
 #include "Editor.h"
-
-#include "lmms_basics.h"
 #include "JournallingObject.h"
+#include "MidiClip.h"
 #include "TimePos.h"
-#include "AutomationClip.h"
-#include "ComboBoxModel.h"
+#include "lmms_basics.h"
+#include "qcolor.h"
```

Comment:
Please use `<QColor>` instead and move this above `<QPushButton>`.

Reply:
Done, I just removed `QColor` since it wasn't needed.